sql/moz-fx-data-shared-prod/search_terms_derived/sanitization_job_metadata_v2/schema.yaml

```diff
@@ -47,3 +47,9 @@ fields:
 - mode: NULLABLE
   name: total_search_terms_appearing_in_allow_list
   type: INTEGER
+- mode: NULLABLE
```

Comment:
I don't think `total_search_terms_appearing_in_allow_list` field needs to be nullable because we've had it since the beginning of the job—if there are none in a given run (unlikely but I guess possible), this should be zero. 

I _do_ think it makes sense for the column we're _adding_, with NULL meaning "this row is from before we started tracking this."

Reply:
By my same logic, though, many the fields that are already in here don't need to be nullable either and are, though, and that's probably my fault 😅 not blocking merge on this! Just a thing that occurred to me.

Reply:
True - I was thinking we might get nulls if the job fails, but checking the [code](https://github.com/mozilla/search-terms-sanitization/blob/main/nightly-job/query_sanitization.py#L266) the counts are all coalesced to 0. Agree this isn't a blocker though!